src/config/configuration.ts

```diff
@@ -37,6 +35,7 @@ const configuration = () => {
   // Logger.log("- Update job groups : " + updateJobGroups);
 
   return {
+    loggerConfigs: [],
```

Comment:
Why don't we load the logger configuration in this class instead doing it in the logger class below?
Please see my comment below on logger.service.ts